src/AvaloniaEdit/TextEditor.cs

```diff
@@ -279,7 +279,7 @@ protected override void OnTemplateApplied(TemplateAppliedEventArgs e)
         /// Gets the scroll viewer used by the text editor.
         /// This property can return null if the template has not been applied / does not contain a scroll viewer.
         /// </summary>
-        internal ScrollViewer ScrollViewer { get; private set; }
+        public ScrollViewer ScrollViewer { get; private set; }
```

Comment:
Why `public`?

Reply:
"Since the ScrollTo functions are still not working I made the ScrollViewer public" I needed this to get an Event if the user scrolls

Reply:
I would rather you fixed the methods :) this should not be public.

Reply:
You can also find the scrollviewer externally by searching the visual tree.

Reply:
ok it is internal again